### PR TITLE
First iteration of actual UI layer

### PR DESCRIPTION
**Pull request by iheart2code**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+
+# A special property that should be specified at the top of the file outside of
+# any sections. Set to true to stop .editor config file search on current file
+root = true
+
+[*]
+# Indentation style
+# Possible values - tab, space
+indent_style = space
```
Comment (iheart2code) [inline .editorconfig:10]: Are you sure we should be using spaces? I'm kinda into tabs lately.

File: ui-server/.gitignore [hunk 2]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+*.sketch
```
Comment (iheart2code) [inline ui-server/.gitignore:25]: Did you make a Sketch file for this??

Comment (bryandowning) [reply ui-server/.gitignore:25]: yep, it's pretty rough though

File: .editorconfig (hunk 1 above)
Comment (bryandowning) [reply .editorconfig:10]: oh god, ew

**Pull request merged**